src/common/options.cc

```diff
@@ -6294,7 +6294,7 @@ std::vector<Option> get_rgw_options() {
     .add_see_also({"rgw_gc_obj_min_wait", "rgw_gc_processor_max_time", "rgw_gc_processor_period", "rgw_gc_max_concurrent_io"}),
 
     Option("rgw_gc_obj_min_wait", Option::TYPE_INT, Option::LEVEL_ADVANCED)
-    .set_default(2_hr)
+    .set_default(300)
```

Comment:
were these changes intended, or just for testing?

per discussion in https://github.com/ceph/ceph/pull/28421#discussion_r303621269, can you please add a `.set_min(30_min)` to this option?

Reply:
These were just for testing, will remove them and add the min value for gc obj min wait